tests/functional/ui/test_alert_text.py

```diff
@@ -79,6 +80,7 @@ def alerts_expected():
 @skipif_hci_client
 @polarion_id("OCS-5509")
 @skipif_disconnected_cluster
+@ui
```

Comment:
Please update the marker name as ui could be misleading, to me it looks like it's an UI test but the definition says something else.

Reply:
@am-agrawa I think this test_runbooks is a UI test - so it's a valid marker here.

It's navigating to specific alerting page and check there.

Reply:
That's exactly what I wanted to understand? Since we are skipping it and not running it, using `ui` marker whose definition is to skip the test on IBM is really confusing to me.
Or did I not understand the implementation?

Reply:
@petr-balogh ^^

Reply:
We sorted out over the direct message in slack